src/Moryx.CommandCenter.Web/src/common/scss/commandcenter.scss

```diff
@@ -9,500 +9,566 @@
- }
-
- .navbar {
-   transition: 0.3s;
-   overflow: hidden;
-   padding: 0;
-   height: inherit;
+.card-header {
+    margin-bottom: 0 !important;
+    height: 50px;
+    padding: 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.navbar {
+    transition: 0.3s;
+    overflow: hidden;
+    padding: 0;
+    height: inherit;
 }
 
 .navbar-default.navbar.navbar-expand-md {
-  background-color: transparent;
+    background-color: transparent;
 }
 
-  .navbar-left.navbar-nav {
+.navbar-left.navbar-nav {
     width: 100%;
     list-style-type: none;
     margin-left: 0;
     margin-top: 0;
     height: inherit;
-  }
+}
 
-  .nav-item {
+.nav-item {
     height: 100%;
     display: flex;
     align-items: center;
     margin-top: 0 !important;
     justify-content: center;
     border-right: 1px solid $gray-200;
-  }
+}
 
-  .nav-item:last-child {
+.nav-item:last-child {
     border: none;
-  }
+}
 
-  .nav-item:hover {
+.nav-item:hover {
     background: $gray-200;
     cursor: pointer;
     color: $gray-600;
-  }
+}
 
-  .active {
+.active {
     background: $gray-300;
     margin-top: 0 !important;
-  }
+}
+
+.nav-tabs {
+    border-bottom: 1px solid #dee2e6;
+
+    .nav-link.active {
+        border-color: #dee2e6 #dee2e6 #fff;
+        color: #495057;
+    }
+
+    .nav-link:hover {
+        border-color: #e9ecef #e9ecef #dee2e6;
+    }
+}
 
-  .navbar-nav-link {
+.table th {
+    border-top: 1px solid #343a40;
+}
+
+.navbar-nav-link {
     color: $gray-800;
     text-decoration: none;
     height: 100%;
     padding: 5px;
-  }
+}
 
-  a.navbar-nav-link {
+a.navbar-nav-link {
     height: 100%;
     display: flex;
     align-items: center;
     justify-content: center;
     padding: 0px 7px;
-  }
+}
 
-  .navbar-nav-link:hover,
-  .navbar-nav-link:focus {
+.navbar-nav-link:hover,
+.navbar-nav-link:focus {
     color: $gray-600;
     text-decoration: none;
-  }
+}
 
-  .navbar-collapse {
+.navbar-collapse {
     height: 50px;
-  }
+}
 
-  .nav-listgroup-item {
+.nav-listgroup-item {
     padding: 0;
     height: 40px;
+    border: 1px solid rgba(0, 0, 0, 0.125);
+    border-top: none;
     border-left: none;
     border-right: none;
-  }
+}
+
+.table {
+    thead th {
+        border-bottom: 2px solid #343a40;
+    }
+
+    td {
+        border-top: 1px solid #343a40;
+        padding: .75rem;
+    }
+}
+
+.table> :not(caption)>*>* {
+    border-bottom: none;
+}
 
 /*
  * Content Panel
  */
 .commandcenter-app-container {
-  display: flex;
-  flex-direction: column;
+    display: flex;
+    flex-direction: column;
 }
 
 .commandcenter-content {
-  margin-top: 20px;
-  height: 100%;
+    margin-top: 20px;
+    height: 100%;
 }
 
 .component {
-  border-radius: 4px;
+    border-radius: 4px;
 }
 
 .card {
-  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
-  transition: 0.3s;
-  overflow: hidden;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    transition: 0.3s;
+    overflow: hidden;
 }
+
 /*
```

Comment:
Does the Dashboard styling refer to the dashboard that you removed?

Reply:
you're right, thanks for the hint. I removed the classed except `no-padding` which is still used outside the dashboard